docs/enterprise/adfs-sso.md

```diff
@@ -22,6 +22,12 @@ StackBlitz is excited to offer SAML-based Single Sign-on (SSO) to organizations
 
 ## StackBlitz Admin Panel Auth Settings Page
 
+:::warning IMPORTANT: 
+Admin accounts should not be made with an individual work email that will be used to create a separate user account later. Should this happen, we recommend the following steps:
+  - Log in as admin
+  - Navigate to https://editor.stackblitz.[COMPANY.COM]/users/edit to change your email address to a different one. It will show the same screen as the changing email/password when first logging in as an admin, but you only need to change your email.
```

Comment:
A slightly different formating should make the link clearer (otherwise it does this weird autolink thing when deployed) 

```suggestion
  - Navigate to `https://editor.stackblitz.[COMPANY.COM]/users/edit` to change your email address to a different one. It will show the same screen as the changing email/password when first logging in as an admin, but you only need to change your email.
```